src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,109 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private int size = 0;
+    private int threshold;
+    private Node<K, V> [] table;
+
+    class Node<K, V> {
+        private final int hash;
+        private K key;
+        private V value;
+        private Node<K, V> next;
+
+        public Node(int hash, K key, V value, Node<K, V> next) {
```

Comment:
There's no need to use public access modifier for the inner class Node and its constructor. By default, it has package-private access, which is enough in this case because this class is not intended to be used outside of the `MyHashMap`.